tests/unit/packages/Teams/js/tests/utils.ts

```diff
@@ -0,0 +1,41 @@
+// Licensed under the MIT License.
+// Copyright (c) Microsoft Corporation. All rights reserved.
+
+import { AdaptiveBotComponent } from 'botbuilder-dialogs-adaptive';
+import {
+  ServiceCollection,
+  noOpConfiguration,
+} from 'botbuilder-dialogs-adaptive-runtime-core';
+import { AdaptiveTestBotComponent } from 'botbuilder-dialogs-adaptive-testing';
+import {
+  ResourceExplorer,
+  ResourceExplorerOptions,
+} from 'botbuilder-dialogs-declarative';
+import { AdaptiveTeamsBotComponent } from '@microsoft/bot-components-teams';
+import path from 'path';
+
+export function makeResourceExplorer(
+  resourceFolder: string,
+  ...botComponents: typeof AdaptiveTeamsBotComponent[]
```

Comment:
Should this be `...botComponents: ...BotComponent[]`?

Reply:
I tried to at first, but it then throws some errors below 
![image](https://user-images.githubusercontent.com/40401643/124839875-835feb80-df3e-11eb-8b23-51b8beac3827.png)

Is there a way around that?

Reply:
Ah, if you need something that "constructs T" you can use `Newable` from `botbuilder-stdlib` or write something like: 
```typescript
...botComponents: Array<new () => BotComponent>,
```

Reply:
That makes sense. Amended.